etl/src/main/java/org/pih/hivmigration/etl/sql/SqlMigrator.java

```diff
@@ -57,8 +66,11 @@ Properties getOracleConnectionProperties() {
     Properties getMysqlConnectionProperties() {
         Properties mp = getMigrationProperties();
         Properties properties = new Properties();
-        properties.put("url", mp.getProperty("mysql.url", "jdbc:mysql://localhost:3308/openmrs?rewriteBatchedStatements=true"));
+        properties.put("url", "jdbc:mysql://" + mp.getProperty("mysql.host", "localhost")
+                + ":" + mp.getProperty("mysql.port", "3308") + "/"
+                + mp.getProperty("mysql.database", "openmrs") + "?rewriteBatchedStatements=true");
```

Comment:
These are the properties that were present in the existing `migration.properties` file